phpdocker/php-fpm/Dockerfile

```
@@ -0,0 +1,10 @@
+FROM phpdockerio/php:8.3-fpm
```

```suggestion
FROM php:8.3-cli-bullseye
```

What if we use the official image?

---

I refactored this know and removed the phpdockerio instance and replaced it with php:8.3-fpm.

I decided against the cli image because then we wouldn't be able to easily run ``exec`` anymore. So I think fpm is just suited better for this use case